sktime/forecasting/moirai_forecaster.py

```diff
@@ -0,0 +1,461 @@
+"""Adapter for using MOIRAI Forecasters."""
+
+from unittest.mock import patch
+
+import pandas as pd
+from skbase.utils.dependencies import _check_soft_dependencies
+
+import sktime.libs.uni2ts
```

Comment:
I would propose to do this import only in an if clauses that checks that all required packages are installed.

Reply:
Hopefully this would also fix the test failures.